master/buildbot/test/unit/test_steps_http.py

```diff
@@ -99,15 +99,19 @@ def test_404(self):
 
     def test_POST(self):
         url = self.getURL("POST")
-        content = "\n<html>\n" \
-                  "  <head><title>405 - Method Not Allowed</title></head>\n" \
-                  "  <body>\n    <h1>Method Not Allowed</h1>\n    <p>Your browser " \
-                  "approached me (at /POST) with the method \"POST\".  I only allow the " \
-                  "methods HEAD, GET here.</p>\n  </body>\n</html>\n"
+        #
```

Comment:
I looked at the code and it is quite intertwined, so you need to look at all of the output or none.  There is no easy way to do a partial compare of the output. 

In the interest of moving forward, I thought that just checking for a 405 status was a reasonable compromise.